public/app/features/plugins/sandbox/utils.ts

```diff
@@ -38,23 +41,47 @@ export function logError(error: Error, context?: LogContext) {
   logErrorRuntime(error, context);
 }
 
-export function isFrontendSandboxSupported({
+export async function isFrontendSandboxSupported({
   isAngular,
   pluginId,
 }: {
   isAngular?: boolean;
   pluginId: string;
-}): boolean {
+}): Promise<boolean> {
```

Comment:
this is a small refactor of the logic that was getting too complicated in a single `if`